cosmos/providers/databricks/notebook.py

```diff
@@ -124,7 +126,10 @@ def monitor_databricks_job(self):
 
     def _get_current_databricks_task(self, runs_api):
         return {
-            x["task_key"]: x for x in runs_api.get_run(self.databricks_run_id)["tasks"]
+            x["task_key"]: x
+            for x in runs_api.get_run(self.databricks_run_id, version=JOBS_API_VERSION)[
```

Comment:
Is this version necessary? Do we need the major and minor digits?

Reply:
Yes, that version `2.1` is necessary as it gives consistent responses. Both major and minor digits are necessary. I checked that version `2.0` sends the json key as `task` whereas `2.1` sends it as `tasks` across single and multi-task runs.